lib/util.js

```diff
@@ -106,6 +106,8 @@ function formatWithOptions(inspectOptions, f) {
             // eslint-disable-next-line valid-typeof
             if (typeof tempNum === 'bigint') {
               tempStr = `${tempNum}n`;
+            } else if (typeof tempNum === 'symbol') {
+              tempStr = 'NaN';
```

Comment:
Maybe store the `typeof` result in a variable for performance, if it even matters.

Reply:
This is inlined by V8 and therefore storing it in an extra variable does not help.